### PR TITLE
Issue#356 Solved Background sheet change

### PR DESCRIPTION
**Pull request by Chaitanya31612**
https://user-images.githubusercontent.com/54992097/106383215-ad8b0b80-63ea-11eb-805b-a8bed662082d.mp4

### CODE REVIEW

File: src/pages/Editor/sections/OutputComponent/Output.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ import { EditContext } from "../../containers/editContext";
 
 const OutputComponent = () => {
   const editContext = useContext(EditContext);
-  const page = require('./ruled1.png');
+  // const page = require('./ruled1.png');
+  const page = require(`./${editContext.pageSrc}`);
  console.log(`${editContext.pageSrc}`);
```
Comment (smaranjitghose) [inline src/pages/Editor/sections/OutputComponent/Output.js:11]: Try to add some other pages as well which may be useful for students

**Review by smaranjitghose — APPROVED** [review]

File: src/pages/Editor/containers/editContext.js [hunk 2]
```diff
@@ -4,8 +4,8 @@ import DomToImage from "dom-to-image";
 export const EditContext = React.createContext();
 
 const EditContextProvider = (props) => {
```
Comment (smaranjitghose) [inline src/pages/Editor/containers/editContext.js:6]: The default page should be the blank page

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/pages/Editor/sections/OutputComponent/Output.js (hunk 1 above)
Comment (Chaitanya31612) [reply src/pages/Editor/sections/OutputComponent/Output.js:11]: Can you provide pages you want me to add?

File: src/pages/Editor/containers/editContext.js (hunk 2 above)
Comment (Chaitanya31612) [reply src/pages/Editor/containers/editContext.js:6]: cool

File: src/pages/Editor/sections/OutputComponent/Output.js (hunk 1 above)
Comment (smaranjitghose) [reply src/pages/Editor/sections/OutputComponent/Output.js:11]: 😅 I have to do a review of the common requirements. Let me check if I have the last year's report for this, I'll send you the images then

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**